components/client/src/test/unit/java/com/hotels/styx/client/OriginsInventoryTest.java

```diff
@@ -85,207 +86,397 @@ public void stop() {
         logger.stop();
     }
 
+    /*
+     * Setting the origins
+     */
     @Test
-    public void addingANewOriginWillAddToActiveSetAndInitiateTheMonitoring() {
-        inventory.addOrigins(ORIGIN);
+    public void setNewOrigins() {
```

Comment:
What is this test trying to prove? Its name doesn't explain.

Reply:
ok